### DISCUSSION TITLE
Question about style training

### DISCUSSION THREAD

**Discussion by sangoi-exe** [TheLastBen/fast-stable-diffusion#2048] [Q&A] [answered]
Would it be possible for me to train in the style of a particular artist in the following way:

1) Take images of the artist and select parts of the body, such as face, hands, feet, etc., crop them into 512x512 and use a common word as a token, such as "face", "hand", "foot" to infuse some of the artist's style into specific keywords.

2) Then I select 10 images (as you suggest) and train them with the unique token I choose. 

Is that possible?

**Comment by TheLastBen** [comment, marked as answer]
no need to separate the parts, just pick 10 images of the style, rename them to one unknown word, then use that word to summon the style, do not add any other word to the instance images names

**Comment by Triceratops9** [comment]
I have a follow up to this question, if I want to make a portrait of a specific person in the style I  just trained, with the unknown word, how would I do this? To clarify, I want to use the artist's style ( I made a .cpkt file with that specific unknown word training on the 10 images of that style with that unknown word.) and then use another .cpkt file I created training on someone specific. How do I go about making the two cpkt files, and then how do I use them in Automatic 1111? Thank you!

**Reply by TheLastBen** [reply]
you need to train a lora for the style

**Reply by Triceratops9** [reply]
Thank you! Can you tell me more about that? How do I train a lora?

**Reply by TheLastBen** [reply]
SDXL will be released in 3 days, you can use my SDXL notebook in Runpod to train any style, it work very well

**Reply by Triceratops9** [reply]
Thank you!
I'm still really confused about how to use LORA's in SD with Automatic 1111. I haven't found a good tutorial that walks me though it. (I'm a total newbie at this.) Are Lora's easier to use in SDXL? I'm also unfamiliar with Runpod. What are the benefits of using that over google colab? (Links to tutorials and videos are great.) Thank you!

**Reply by TheLastBen** [reply]
you simply put the lora files in the folder models/lora in A1111 main folder, then load the list of the loras using the webui.

Runpod is cheaper and faster then colab, I have the notebooks there https://www.runpod.io/console/gpu-cloud?template=runpod-stable-unified, remember to stop the pod after using it just like disconnecting from google colab
